### ISSUE TITLE
Accuracy_

### ISSUE THREAD

**Issue by yohehe** [yohehe/pytorch_temp#5]

**Comment by yohehe** [comment]
#上記を変換する
from sklearn.metrics import precision_score,recall_score,f1_score

#評価指標を出力
def OutputAccuracy(inputs,labels,threshold=0.5,evaluation="f1",average="macro"):
    
    """
    inputs:指定したmodelの出力の確認を
    thredshold:閾値の設定
    evaluation:評価指標 f1,recall,precisio
    average=macroなどの指定を
    """
    
    #sigmoid->cpu
    input_sigmoid=torch.sigmoid(inputs).detach().cpu()
    #print(input_sigmoid)
    
    #label:->cpu
    labels=labels.detach().cpu().numpy()
    
    #[batch_num,targets]の出力になっているため、
    
    #thresholdでbool->[0,1]に変換する
    input_thres=(input_sigmoid>=threshold).type(torch.float)
    input_thres=input_thres.numpy()
    
    if evaluation=="precision":
        accuracy=precision_score(input_thres,labels,average=average)
    elif evaluation=="f1":
        accuracy=f1_score(input_thres,labels,average=average)
    elif evaluation=="recall":
        accuracy=recall_score(input_thres,labels,average=average)
    return accuracy
